core/tests/fireproof/attachable-subscription.test.ts

```
@@ -0,0 +1,500 @@
+            return { ...tdb, subscriptionPromise };
+          }),
+      );
+
+      // Wait for initial sync to complete (nothing to sync yet)
+      await sleep(1000);
+
+      // Now write NEW data to databases - this WILL trigger subscriptions ✅
+      // This is the key difference: NEW writes vs EXISTING data sync
+      const keys = (
+        await Promise.all(
+          dbs.map(async (db, index) => {
```

ESLint

[error] 'index' is defined but never used. Allowed unused args must match /^_/u.

(@typescript-eslint/no-unused-vars)